Example/Podfile.lock

```diff
@@ -0,0 +1,23 @@
+PODS:
+  - OSSSpeechKit (0.3.2)
```

Comment:
Podfile.lock should be ignored. I have this as a global ignore but it's probably worthwhile adding to the .gitignore file for this repo. I'm glad this picked it up.

Reply:
No sure if I have no added it correctly or not.

Reply:
Close - Example/Podfile.lock

Reply:
@AppDevGuy Sorry for being a complete dummy. You mean:
<img width="538" alt="Screenshot 2021-06-09 at 10 37 50 PM" src="https://user-images.githubusercontent.com/45777358/121365150-5df9b500-c973-11eb-985b-1b89ede083b5.png">

Reply:
Not a dummy at all - it's ok to ask questions. 

That looks like you have it.

Reply:
@AppDevGuy Should be added correctly now.